Alignment/MillePedeAlignmentAlgorithm/plugins/MillePedeDQMModule.cc

```diff
@@ -30,8 +30,14 @@
 /*** Thresholds from DB ***/
 #include "CondFormats/DataRecord/interface/AlignPCLThresholdsRcd.h"
 
+/*** Necessary Framework infrastructure ***/
+#include "FWCore/Framework/interface/ProcessBlock.h"
+#include "DataFormats/Alignment/interface/AlignmentToken.h"
+
 MillePedeDQMModule ::MillePedeDQMModule(const edm::ParameterSet& config)
-    : mpReaderConfig_(config.getParameter<edm::ParameterSet>("MillePedeFileReader")) {}
+    : mpReaderConfig_(config.getParameter<edm::ParameterSet>("MillePedeFileReader")) {
+  consumes<AlignmentToken, edm::InProcess>(edm::InputTag("SiPixelAliPedeAlignmentProducer"));
```

Comment:
it's not very clear to me, how this input tag should be set. Is it the name of the module producing the `AlignmentToken` in the event? I could not infer it from the examples in `cmssw`.

Reply:
It should be set to the label of the module that produces the `AlignmentToken` in the `ProcessBlock`. From the issue #31381 the label looks correct here. By eye this PR looks ok, so if it doesn't work I'll need to look myself.

Reply:
It is the module label of the module producing the `AlignmentToken` in the `EndProcessBlock`.

Reply:
My suggestion is to make the `InputTag` value a configuration parameter of the module rather than hard coding it.

Reply:
Thanks to both, I took the suggestion about making the `InputTag` configurable, while applying the code-checks.